src/ui/ons/partials/ONSRadio.svelte

```diff
@@ -1,15 +1,25 @@
 <script>
   import { getContext } from "svelte";
 
-  export let id;
-  export let value;
-  export let checked = false;
+  export let id, value, radioValue, onChange;
   let name = getContext("name");
+  let userSelection;
 </script>
 
 <p class="ons-radios__item">
   <span class="ons-radio">
-    <input type="radio" {id} class="ons-radio__input ons-js-radio" bind:value {name} />
+    <input
+      type="radio"
+      {id}
+      class="ons-radio__input ons-js-radio"
+      {value}
+      bind:group={userSelection}
+      {name}
+      on:change={() => {
+        radioValue = userSelection;
+        onChange(radioValue);
+      }}
+    />
```

Comment:
I don't know why but for some reason the horizontal space between radio buttons has disappeared on my machine - before we merge see if you can replicate
![image](https://user-images.githubusercontent.com/6823289/142016761-471ec933-08a5-4cfe-8e94-f80664dcd68d.png)

Reply:
I fixed this with @designorant  help by just updating the `<ONSRadio/>` component as per the DS example. I've changed the wrapper `<p>`element to a `<span>` and added a `<br/>` element at the end just like in the [ DS example](https://ons-design-system.netlify.app/components/radios/).